docs/content/quickstart/_index.md

```diff
@@ -116,6 +116,7 @@ History:
 ## Upgrade the Installation
 
 To upgrade the resources managed by the bundle, use `porter upgrade`.
+### Upgrade using a version tag
```

Comment:
I just realized that we aren't taking advantage of Porter's --version flag for upgrade.

A more simple way to upgrade using a tag is `porter upgrade myinst --version 1.2.3` and then you don't need to respecify the bundle.

Do you want to update the example here? Or I can open a follow-up PR after yours is merged.

Reply:
Oh nice, I can update the example here